src/demo-app/typography/typography-demo.html

```diff
@@ -0,0 +1,15 @@
+<h1 class="mat-h0">Lorem ipsum dolor sit amet.</h1>
+<h1 class="mat-h1">Lorem ipsum dolor sit amet, consectetur adipisicing.</h1>
+<h2 class="mat-h2">Lorem ipsum dolor sit amet, consectetur adipisicing.</h2>
+<h3 class="mat-h3">Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+<h4 class="mat-h4">Lorem ipsum dolor sit amet, consectetur adipisicing.</h4>
+<h5 class="mat-h5">Lorem ipsum dolor sit amet, consectetur adipisicing.</h5>
+<h6 class="mat-h6">Lorem ipsum dolor sit amet, consectetur adipisicing.</h6>
+
+<div class="mat-body">
+  <p>Lorem ipsum dolor sit amet, <span class="mat-body-strong">consectetur</span> adipisicing elit. Iure voluptatem amet facilis rerum non dolore repellendus ab. Assumenda nisi perspiciatis odit illum voluptatem expedita laborum! Debitis nisi eius, ratione nostrum velit dignissimos molestias saepe, esse facilis blanditiis, labore optio. Accusantium, nihil illo est beatae nobis expedita animi libero, laboriosam excepturi consequatur eaque, ab repudiandae.</p>
```

Comment:
optional: I like adding something more meaningful / whimsical for demos, up to you